### ISSUE TITLE
Update wiki: configure: error: gtkmm-3.0 development library not found.

### ISSUE THREAD

**Issue by Cheese-oOo** [meduzapat/LEDSpicerUI#26]
Compile error fix:

After ./configure CXXFLAGS='-g0 -O2' reports :
configure: error: gtkmm-3.0 development library not found.

Run command to install dependency:
sudo apt install libgtkmm-3.0-dev

**Comment by meduzapat** [comment]
That is correct. You need gtlmm 3 (libgtkmm-3.0-dev) as well as tinyxml 2 (libtinyxml2-dev). Please note that this is a work-in-progress branch and is still very incomplete. I will be sending more changes later this week. In my next push, I will include input plugins and most of the common features. Only animations are left!
